### ISSUE TITLE
Slick-track width increasing in small screens

### ISSUE THREAD

**Issue by Licorb** [kenwheeler/slick#1232]
I'm having this problem (please, see the image). It's happening on every browser I tested (Chrome 41, FF 37, Opera 28 and IE 11.0). So, when I load the page on mobile devices or resize it the slick-track (and my div's) width goes crazy and start to increase to very large numbers.

I've tried to set a maximum width in the container div and elements, but with no effect
in the problem. 
![issue-slick](https://cloud.githubusercontent.com/assets/5069075/7138262/87ecc972-e295-11e4-9309-d9f3d35019dc.jpg)

**Comment by heppstux** [comment]
+1 same problem here.

**Comment by kenwheeler** [comment]
Are you using flexbox or tables in there?

**Comment by heppstux** [comment]
Hey Ken,
no flexbox, no tables – just normal divs in my case.

**Comment by kenwheeler** [comment]
Can you make a jsFiddle, isolating the issue without the possibility of external style overrides?

**Comment by heppstux** [comment]
Yea can try.

**Comment by Licorb** [comment]
After some tests I solved the problem. Sorry but it has nothing to do with slick, the problem was in the framework I'm using (MaterializeCss).

**Comment by petrleocompel** [comment]
Did you solve that problem ?
I'm having same troubles with bootstrap :/

**Comment by simeydotme** [comment]
@Licorb it's usually a good idea to post your solutions so people like @petrleocompel can find them in the future :stuck_out_tongue_closed_eyes:

**Comment by Licorb** [comment]
Well, in my case the problem was with the way MaterializeCSS framework calculates columns and rows; slick was breaking these calculations in small screens(I don't know why). The solution in my case was to get rid of the class controlling small screen's columns, and to do this without messing up with my entire layout, I created a specific class for the slide. I think this is pretty specific and don't know if is the proper solution.

in any case: 
In grid.scss I copied from line 119 to 186 (col class) and pasted changing the class to col-slick and changing '@media #{$medium-and-up}' for '@media #{$medium-and-down}'
